core-java-modules/core-java-lang-operators-2/src/test/java/com/baeldung/operators/ConditionalUnitTest.java

```diff
@@ -0,0 +1,55 @@
+package com.baeldung.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class ConditionalUnitTest {
+
+    @Test
+    void whenUseTheOperatorConditionalIncluive() {
+        int years = 25;
+        boolean driverLicence = true;
+        String result = "";
+        if (years >= 21 && driverLicence) {
+            result = "Successful Candidate.";
+        } else {
+            result = "Failed Candidate.";
+        }
+        assertEquals(result, "Successful Candidate.");
+
+    }
+
+    @Test
+    void whenUseTheOperatorConditionalInclusiveWithTernary() {
+        int years = 25;
+        boolean driverLicence = true;
+        String result = (years >= 21 && driverLicence) ? "Successful Candidate." : "Successful Candidate.";
```

Comment:
The result in both cases is the same: Successful Candidate.

Reply:
The motivation  in that case is to explain about how to write the same code using the ternary mode.

Reply:
What I mean is that the previous examples show a Successful Candidate and Failed Candidate. for the else branch. Here, both results are the same: Successful Candidate